config/install/field.field.node.profile.field_landing_page_c_secondary.yml

```diff
@@ -3,7 +3,6 @@ status: true
 dependencies:
   config:
     - block_content.type.campaign
-    - block_content.type.campaign_with_video
```

Comment:
this has been removed from tide_landing_page as a part of a clean up request. This component doesn't get used